app/main.py

```diff
@@ -1,13 +1,24 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+    return {
+        "line": error["line_number"],
+        "column": error["column_number"],
+        "message": error["text"],
+        "name": error["code"],
+        "source": "flake8"
+    }
 
 
 def format_single_linter_file(file_path: str, errors: list) -> dict:
-    # write your code here
-    pass
+    return {
+        "errors": [format_linter_error(er) for er in errors],
+        "path": file_path,
+        "status": "failed"
+        if len([format_linter_error(er) for er in errors]) != 0
+        else "passed",
+    }
 
 
 def format_linter_report(linter_report: dict) -> list:
-    # write your code here
-    pass
+    return [format_single_linter_file(path, linter_report[path])
```

Comment:
Well done but the braces must be located in one of two options: open and start with the text or have a line break between the text.
Here is an example:
https://github.com/mate-academy/py-linter-formatter/blob/master/checklist.md